pkg/controllers/elasticsearch/actions/create_nodepool.go

```diff
@@ -226,14 +254,12 @@ func elasticsearchPodTemplateSpec(controllerName string, c *v1alpha1.Elasticsear
 								Path: "/",
 							},
 						},
-						InitialDelaySeconds: int32(60),
-						PeriodSeconds:       int32(10),
-						TimeoutSeconds:      int32(5),
-					},
-					Resources: apiv1.ResourceRequirements{
-						Requests: np.Resources.Requests,
-						Limits:   np.Resources.Limits,
+						InitialDelaySeconds: 240,
```

Comment:
This figure changed from 60 to 240. How come?

Reply:
During testing I was having issues where the liveness probe wasn't passing within 60s, causing crash loops.

Really, we should probably change the Elasticsearch Pilot liveness probe to *just* check that Pilot is running (i.e. Pilot should always return true, and not attempt to query ES itself)

Reply:
I think that's what it does already.

```
func (p *Pilot) LivenessCheck() probe.Check {
	return probe.CombineChecks(
		func() error {
			return nil
		},
	)
}
```